medical/security/medical_security.xml

```diff
@@ -7,40 +7,38 @@
     </record>
 
     <record id="group_medical_user" model="res.groups">
-        <field name="name">Medical User</field>
```

Comment:
They have to be, otherwise the group will be inheriting from a group that hasn't been created yet via the implied_ids